### PR TITLE
Add exe 14 files

### PR DESCRIPTION
**Pull request by Reyes-PaulAngelo**
add my exe 14 files

### CODE REVIEW

File: Reyes_PaulAngelo/works/Countries.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <link rel="stylesheet" href="./css_country.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Countries of the World</h1>
+      <input
+        type="text"
+        id="countryInput"
+        placeholder="Enter country name..."
+      />
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/Countries.html]: Remove unnecessary breaking of lines. Place line breaks in proper places.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27

File: Reyes_PaulAngelo/works/Countries.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      <button onclick="searchCountry()">Search</button>
+      <div class="country-details" id="countryDetails"></div>
+      <div class="same-region-countries" id="sameRegionCountries"></div>
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/Countries.html]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_PaulAngelo/works/script.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+  function addComment(name, text, date) {
+    const commentElement = document.createElement("p");
+    commentElement.innerHTML =
+      "Name: " +
+      name +
+      "<p>Comment: " +
+      text +
+      "<p><small>" +
+      date.toLocaleString() +
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js]: I recommend following this format instead:
Additionally, you forgot to include the closing tag for the `<p>`
```
         "Name: " + comment.name +
        "<p>Comment: " + comment.text +
        "<p><small>" +comment.date.toLocaleString()
```

File: Reyes_PaulAngelo/works/script.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+      "</small>";
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js]: Where is the opening tag for this?

File: Reyes_PaulAngelo/works/script.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    commentsContainer.innerHTML = "";
+    comments.forEach(function (comment) {
+      const commentElement = document.createElement("p");
+      commentElement.innerHTML =
+        "Name: " +
+        comment.name +
+        "<p>Comment: " +
+        comment.text +
+        "<p><small>" +
+        comment.date.toLocaleString() +
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js]: I recommend following this format: 
Additionally, you forgot to include the closing tag for the `<p>`
```
      "Name: " + name +
      "<p>Comment: " + text +
      "<p><small>" + date.toLocaleString() +
```

File: Reyes_PaulAngelo/works/script.js [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+        ${country.region ? country.region : "N/A"}</p>
+        <p><strong>Languages:</strong> 
+        ${
+          country.languages
+            ? Object.values(country.languages).join(", ")
+            : "N/A"
+        }</p>
+        <p><strong>Timezones:</strong> 
+        ${country.timezones ? country.timezones.join(", ") : "N/A"}</p>
+      `;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js]: When splitting the elements into multiple lines, make sure that the continuation line is indented.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_11

File: Reyes_PaulAngelo/works/script.js [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+      document.getElementById("countryDetails").innerHTML = details;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_PaulAngelo/works/script.js [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+    .then(function (regionData) {
+      let region = regionData[0].region;
+      let sameRegionCountriesList = regionData
+        .map(function (c) {
+          return `
+          <div class="same-region-country">
+            <img src="${c.flags.svg}" alt="Flag of ${c.name.common}" width="50">
+            <p>${c.name.common}</p>
+          </div>
+        `;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js]: When splitting the elements into multiple lines, make sure that the continuation line is indented.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_11

File: Reyes_PaulAngelo/works/script.js [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+        })
+        .join("");
+      document.getElementById("sameRegionCountries").innerHTML = `
+        <h2>Countries in the Same Region (${region})</h2>
+        <div>${sameRegionCountriesList}</div>
+      `;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js]: You can have this on the same line.

`<div>${sameRegionCountriesList}</div>`;` <please dont forget the backtick>

File: Reyes_PaulAngelo/works/script.js [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+    })
+    .catch(function (error) {
+      console.error("Error fetching data:", error);
+      document.getElementById("countryDetails").innerHTML =
+        "<p>An error occurred: " + error.message + "</p>";
+      document.getElementById("sameRegionCountries").innerHTML = "";
+    });
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/script.js:156]: When you use try…catch statement, add only the statement that will throw an error. Mover other statements outside the try…catch block.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g2651733a4ec_0_0

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]
Place all your assets into a common folder so it won't clutter your project directory.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g26e9a7bd30a_1_0

**Review by Villa-VillaDiana — APPROVED** [review]
You can now merge this :))

**Comment by Reyes-PaulAngelo** [comment]
thank you

**Pull request merged**